api/v1beta1/agentserviceconfig_types.go

```diff
@@ -109,6 +109,12 @@ type AgentServiceConfigSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="List of container registries without authentication"
 	// +optional
 	UnauthenticatedRegistries []string `json:"unauthenticatedRegistries,omitempty"`
+	// OSImageCACertConfigMapRef is a reference to a config map containing a certificate authority certificate
+	// this is an optional certificate to allow a user to add a certificate authority for a HTTPS source of images
+	// this certificate will be used by the assisted-image-service when pulling OS images.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="OS Image CA Cert ConfigMap reference"
+	// +optional
+	OSImageCACertConfigMapRef *corev1.LocalObjectReference `json:"osImageCACertConfigMap,omitempty"`
```

Comment:
I don't think we need to say `ConfigMap` here.
We don't in `MirrorRegistryRef` and that's also a reference to a `ConfigMap`

Reply:
Also the json serialization name should match the field name whenever possible just to reduce confusion.